### PR TITLE
Add example for cloudkms

### PR DESCRIPTION
**Pull request by aljim**

### CODE REVIEW

File: examples/v2/cloudkms/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## Deploy the template
+
+Use `config.yaml` to deploy this example template.
+
+When ready, deploy with the following command:
+
+    gcloud deployment-manager deployments create YOUR_DEPLOYMENT_NAME --config config.yaml
+
+CloudKMS KeyRings and CryptoKeys cannot be deleted, so you need to abandon the resources.
```
Comment (krwenholz) [inline examples/v2/cloudkms/README.md]: Can they be destroyed?

File: examples/v2/cloudkms/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    
+    gclou deployment-manager deployments delete YOUR_DEPLOYMENT_NAME --delete-policy=ABANDON
```
Comment (krwenholz) [inline examples/v2/cloudkms/README.md]: s/gclou/gcloud

File: examples/v2/cloudkms/README.md (hunk 1 above)
Comment (aljim) [reply examples/v2/cloudkms/README.md]: No they can't

**Comment by aljim** [comment]
Fixed typo, removed binary file, added gitignore

**Pull request merged**